### ISSUE TITLE
Award Points Manually

### ISSUE THREAD

**Issue by crookedneighbor** [Commander-Spellbook/website-v2#195]
Acceptance Criteria:

Depends on #194 

- [ ] [create permission](https://github.com/Commander-Spellbook/website-v2/blob/eb29c43351011778ba51800acf2d2d964cecef11/firebase-functions/src/shared/constants.ts) that will allow certain users to manually award/subtract points 
- [ ] create api route that allows user award points to another user
  - [ ] restrict this to users with award points permission
  - [ ] prevent users from awarding points to themselves
  - [ ] require award amount to be a full number
  - [ ] require award amount to be greater than 0
  - [ ] set limit for how much can be awarded
- [ ] create UI for awarding points

**Comment by crookedneighbor** [comment]
No longer relevant as the backend system is happening in a separate app now.
